op-service/txmgr/txmgr.go

```diff
@@ -104,6 +108,10 @@ func (m *SimpleTxManager) IncreaseGasPrice(ctx context.Context, tx *types.Transa
 
 	var gasTipCap, gasFeeCap *big.Int
 
+	// new = old * (100 + priceBump) / 100
+	a := big.NewInt(100 + priceBump)
+	b := big.NewInt(100)
+
```

Comment:
These technically don't need to be allocated on every invocation of this function, they could be `var`s at the top of the file